addons/udes_stock/models/stock_picking_type.py

```diff
@@ -67,3 +73,17 @@ def can_handle_multiple_users(self):
         """
         self.ensure_one()
         return self.u_multi_users_enabled
+
+    def write(self, vals):
+        """
+        Purpose: This will remove the sequence code from the list of values to be updated. This is to prevent the sequences
+        for each picking type from being updated with hardcoded values. sequence_code is only relevant to the old odoo prefixes
+        and not the UDES ones, hence a warning message is also displayed.
+        """
+        if "sequence_code" in vals:
+            _logger.warning(
```

Comment:
Does this message need to be internationalised?

Reply:
Some instance of the logger.warning message are internationalised and some are not. I am not sure if there is a rule for internationalising strings it that I am not aware of, but I will internationalise it here.